starter-code/server.js

```diff
@@ -1,33 +1,21 @@
 'use strict';
 
-// TODO: Initialize your project using NPM to create and populate a package.json file - npm init
-// TODO: Require the Express package that you installed via NPM, and instantiate the app
 const express = require('express');
-
-// Remember to install express, and be sure that it's been added to your package.json as a dependency- npm install --save express
-
-// There is also a package here called body-parser, which is required in for use in a new route - npm install --save body-parser
-// Be sure to install that and save it as a dependency after you create your package.json.
+const app = express();
 const bodyParser = require('body-parser').urlencoded({extended: true});
 const PORT = process.env.PORT || 3000;
 
-// TODO: Include all of the static resources as an argument to app.use()
-app.use((express.static('./public'));
+app.use(express.static('./public'));
 
-// TODO: Write a new route that will handle a request and send the new.html file back to the user
-// app.get('/new.html', function(request,response){
-  app.get('/route', (req, res) =>
-  res.send('this is new html', { root: '.'});
-)
+app.get('/route', (req, res) =>
+  res.send('this is new html', { root: '.'})
```

Comment:
This route should send the actual contents of the `new.html` file. Take a look at the [express docs ](https://expressjs.com/en/4x/api.html#res)to find the method you need.